tensorflow/contrib/avro/BUILD

```diff
@@ -0,0 +1,126 @@
+package(default_visibility = ["//tensorflow:__subpackages__"])
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+load("//tensorflow:tensorflow.bzl", "tf_custom_op_py_library")
+load("//tensorflow:tensorflow.bzl", "tf_custom_op_library")
+load("//tensorflow:tensorflow.bzl", "cc_header_only_library")
+load("//tensorflow:tensorflow.bzl", "tf_gen_op_wrapper_py")
+load("//tensorflow:tensorflow.bzl", "tf_gen_op_libs")
+
+tf_custom_op_py_library(
+    name = "avro",
+    srcs = glob(["python/*.py"]),
+    dso = [":python/_avro_record_dataset.so",
+           ":python/_parse_avro_record.so",
+           ":python/libavro.so"
+    ],
+    kernels = [
+        ":avro_record_dataset_op_lib",
+        ":parse_avro_record_op_lib",
+    ],
+    deps = [
+         ":avro_record_dataset",
+         ":parse_avro_record",
+    ],
+    srcs_version = "PY2",
```

Comment:
Changed. This was quite some effort because avro releases `avro` and `avro-python3` as separate pip packages and their API differ slightly.  I've worked around these differences by introducing wrappers in `avro_serialization.py` and dropping explicit argument naming (bad because sometimes it's reader_schema and sometimes writer_schema)